architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
-xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
-container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+An _image stream_ is a mapping between tags in {product-title} objects and 
+xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker image] images. 
```

Comment:
"An image stream and its associated tags provide an abstraction for referencing Docker images from within OpenShift."